Tests/TurboNavigatorTests/TurboNavigatorTests.swift

```diff
@@ -89,6 +89,22 @@ final class TurboNavigatorTests: XCTestCase {
         assertVisited(url: proposal.url, on: .modal)
     }
 
+    func test_default_modal_default_presentsModal_default() {
+        let proposal = VisitProposal(context: .modal, modal: .default)
+        navigator.route(proposal)
+
+        XCTAssertEqual(modalNavigationController.modalPresentationStyle, .pageSheet)
```

Comment:
Let's add a comment as to why we are testing for `.pageSheet` and that behavior might be different for non-iPhone devices.